frontend/packages/dev-console/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    rules: {
+        'react/no-unescaped-entities': 0,
+    },
+};
```

Comment:
Do we need it?  We can avoid using apostrophes.

Reply:
removed :)